docs/reference/running.md

```diff
@@ -18,12 +18,15 @@ Starting with Ambassador 0.35, we support running Ambassador as non-root. This i
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: ambassador
 spec:
   replicas: 1
+  selector:
```

Comment:
There are a _lot_ of places in this PR where you add `selector`s to a spec. I think those should be a separate PR, unless I'm missing something.

Reply:
The selector is required in the `apps/v1` version of `Deployment`